packages/ag-charts-community/src/util/document.ts

```diff
@@ -0,0 +1,7 @@
+export function onContentLoaded(handler: CallableFunction) {
+    if (typeof document === 'undefined' || document.readyState === 'complete') {
+        handler();
```

Comment:
I'm still a bit concerned about calling this immediately. This was originally added to fix a jump where it would start the animation at 10% or more into the animation on load.

The `DOMContentLoaded` listener is not fired in an iframe, so to get the same effect I put it in that timeout to break it out and run asynchronously, i.e. once the browser is not busy.

This ensured it always started the animation at 0%.

Reply:
I'm not particularly worried about this, we can always fix that later if we find it being an issue. So feel free to merge.

Reply:
I've talked to @alantreadway about it, he thinks that a fix that was added about a 6 weeks ago should have fixed that problem as well. I'll do some testing to make sure that's really the case, and if so I'll remove this code from here. (said fix is in `sizeMonitor.ts`)